data/fedora-update.yaml

```diff
@@ -449,18 +455,73 @@ python-django-dpaste:
       Project is retired, replaced by not-yet-packaged `dpaste`
     links:
       homepage: https://pypi.python.org/pypi/django-dpaste/0.2.4
+python-django-dynamite:
+    dead-upstream: true
+    links:
+      homepage: https://github.com/z4r/django-dynamite
+    note: |
+      Not maintained since 2009
```

Comment:
You are right, it is a typo. The latest activity was there in 2012. Thanks for your check. Please could you make the small correction in the note directly?